code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -176,7 +176,7 @@
 	if(HAS_TRAIT(target, TRAIT_HUSK))
 		if(advanced)
 			if(HAS_TRAIT_FROM(target, TRAIT_HUSK, BURN))
-				render_list += "<span class='alert ml-1'>Subject has been husked by [conditional_tooltip("severe burns", "Tend burns and apply a de-husking agent, such as [/datum/reagent/medicine/c2/synthflesh::name].", tochat)].</span><br>"
+				render_list += "<span class='alert ml-1'>Subject has been husked by [conditional_tooltip("severe burns", "Tend burns and then apply a de-husking agent, such as 60u of pure [/datum/reagent/medicine/c2/synthflesh::name].", tochat)].</span><br>"
```

Comment:
Combined with the other feedback I think adding more information to this is unnecessary

Reply:
Okay.